### PR TITLE
Updated travisfile + added missing builds

### PR DESCRIPTION
**Pull request by madebr**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -38,6 +38,13 @@ def config_options(self):
     def requirements(self):
         self.requires("libtirpc/1.1.4@bincrafters/stable")
 
+    def system_requirements(self):
+        if tools.os_info.with_yum:
+            packages = ["gettext-devel"]
```
Comment (uilianries) [inline conanfile.py]: I've started a recipe for [gettext](https://github.com/bincrafters/conan-gettext_installer) some time ago. Seems like be time to revisit that recipe.

Comment (madebr) [reply conanfile.py]: I cannot reproduce the error that made me add this requirement.
So I've removed the extra requirement.

Comment (madebr) [reply conanfile.py]: I think conan-gettext should be a hybrid project: library + installer.

**Comment by SSE4** [comment]
@madebr `gettext` should be ready for use. feel free to add it into `libnsl`. also, feel free to create PR adding `installer` to the `gettext` build.

**Comment by uilianries** [comment]
@madebr I think your forced push overwrote all changes related to Travis file.

**Comment by madebr** [comment]
@uilianries I just checked and the travis file is up to data: all linux gcc and clang compilers present.
https://github.com/bincrafters/conan-libnsl/pull/2 has the travis changes.

**Comment by uilianries** [comment]
Nice! :D

**Comment by uilianries** [comment]
let's wait https://github.com/bincrafters/conan-libnsl/pull/3 first. Your PR requires GCC9. I'm waiting https://travis-ci.com/bincrafters/conan-libtirpc/builds/120223500

**Pull request merged**